test/uk/gov/hmrc/customs/emailfrontend/services/CustomsDataStoreServiceSpec.scala

```diff
@@ -55,7 +57,8 @@ class CustomsDataStoreServiceSpec extends SpecBase with BeforeAndAfterEach {
     protected val service = new CustomsDataStoreService(mockConnector)
     protected val enrolmentIdentifier = EnrolmentIdentifier("EORINumber", "GB123456789")
     protected val email = "abc@def.com"
-    protected val dateTime = DateTime.parse("2021-01-01T11:11:11.111Z")
+    protected val dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSX")
```

Comment:
Please move this formatter (if it is only used in Tests) in a new TestData class under test.utils package and use that here and at other places as well